projects/ngu-carousel/src/lib/ngu-carousel/ngu-carousel.component.ts

```diff
@@ -59,12 +59,26 @@ export class NguCarousel<T> extends NguCarouselStore
   private withAnim = true;
   activePoint: number;
   isHovered = false;
+  private inputs: NguCarouselConfig;
+  // tslint:disable-next-line:no-input-rename
+  @Input('inputs')
+  set publicInputs(inputs: NguCarouselConfig) {
+    this.inputs = inputs;
+  }
+  private carouselLoad = new EventEmitter<any>();
+  // tslint:disable-next-line:no-input-rename
+  @Output('carouselLoad')
+  get publicCarouselLoad(): EventEmitter<any> {
+    return this.carouselLoad;
+  }
 
-  @Input() private inputs: NguCarouselConfig;
-  @Output() private carouselLoad = new EventEmitter();
-
+  private onMove = new EventEmitter<NguCarousel<T>>();
   // tslint:disable-next-line:no-output-on-prefix
-  @Output() private onMove = new EventEmitter<NguCarousel<T>>();
+  // tslint:disable-next-line:no-input-rename
+  @Output('onMove')
```

Comment:
@Output() onMove = new EventEmitter<NguCarousel<T>>();

Reply:
this works